src/main/webapp/app/shared/alert/alert-error.component.ts

```diff
@@ -70,7 +70,7 @@ export class JhiAlertErrorComponent implements OnDestroy {
                     break;
 
                 default:
-                    if (httpErrorResponse.error !== '' && httpErrorResponse.error.message) {
+                    if (httpErrorResponse.error && httpErrorResponse.error.message) {
```

Comment:
This was a small issue that I discovered: When httpErrorResponse.error was null this caused an issue.